sources/camunda-docker-compose-core.yml

```diff
@@ -63,8 +63,19 @@ services:
       - zeebe
       - elasticsearch
 
+  connectors: # https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
+    image: camunda/connectors-bundle:${CAMUNDA_CONNECTORS_VERSION:-0.1.2}
+    container_name: connectors
+    environment:
+      - ZEEBE_ADDRESS=zeebe:26500
+      - ZEEBE_INSECURE=true
+    networks:
+      - camunda-platform
+    depends_on:
+      - zeebe
+
   elasticsearch: # https://hub.docker.com/_/elasticsearch
-    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTIC_VERSION:-7.17.0}
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTIC_VERSION:-7.17.5}
```

Comment:
why did camunda need elastic search again

Reply:
As per the previous PR conversation, I believe elastic search is required for the prometheus event monitoring. 
@andrewsignori-aot correct me if that is wrong.

Reply:
This docker-compose comes from the below source. We are not changing it just consuming it unless there is something really needed to be changed. 
https://github.com/camunda/camunda-platform/tree/8.1.2

Reply:
mm...okay .. so for monitoring and visualization Prometheus (like grafana) is used, which uses elastic search, and for elastic search monitoring it uses kibana..

Reply:
Right now the only applications that I really concerned about and using are Zeebe and Operate while running on my local machine. While in the cloud we also have "Optimize" for performance improvements, but it was not explored yet.

![image](https://user-images.githubusercontent.com/61259237/197278051-ba6793ec-6f53-4897-bb73-c7c04cf9218d.png)